Formula/i/inform6.rb

```diff
@@ -7,6 +7,11 @@ class Inform6 < Formula
   license "Artistic-2.0"
   head "https://gitlab.com/DavidGriffith/inform6unix.git", branch: "master"
 
+  livecheck do
+    url "http://ifarchive.org/if-archive/infocom/compilers/inform6/source/"
+    regex(/href=.*?inform[._-]v?(\d+(?:\.\d+)+-r\d+)\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?inform[._-]v?(\d+(?:[.-]\d+)+(?:[._-]r\d+)?)\.t/i)
```

This suggestion will match the current format (e.g., `6.41-r6`) as well as the old formats (e.g., `6.33-6.12.1`, `6.34-6.12.4-1.`) and will also continue to match if they ever omit the `-r#` suffix in the future. A little more flexibility shouldn't hurt in this case, though it may help to keep this from breaking if they ever change the version format again.